doc/tutorials/lris_howto.rst

```diff
@@ -0,0 +1,485 @@
+
+.. include:: ../include/links.rst
+
+.. _lris_howto:
+
+=================
+Keck-LRIS HOWTO
+=================
+
+Overview
+========
+
+This doc goes through a full run of PypeIt on one of our :ref:`lris`
+datasets, specifically the ``keck_lris_red_mark4/multi_600_10000_slitmask``
+dataset. These are Keck LRIS RED multi-slit observations taken with the new
```

Comment:
Forgive this request, but might we do a `longslit` too?  I realize that multi-slit is "harder", but I worry that there are *many* observers who only use the longslit and would get confused by multi-slit.

Reply:
@profxj in the interest of time (for my mental sanity) I decided not to write another howto for longslit, but instead I added several comments here to point out the differences. Let me know what you think.

Reply:
thx!